app/instrumentation-tests/src/org/commcare/utils/InstrumentationUtility.kt

```diff
@@ -283,6 +287,13 @@ object InstrumentationUtility {
         Espresso.closeSoftKeyboard()
     }
 
+    /**
```

Comment:
Empty java doc, either add a description or remove the comments.

Reply:
descriptions added.